google/cloud/firestore_v1/gapic/firestore_client.py

```diff
@@ -28,7 +28,6 @@
 import google.api_core.gapic_v1.routing_header
 import google.api_core.grpc_helpers
 import google.api_core.page_iterator
-import google.api_core.path_template
```

Comment:
This is a potentially breaking change:  it relies on sombebody *else* to have imported it before we use it below.  See: https://github.com/googleapis/python-storage/issues/226

@software-dov Is this a known bug in the generator?

Reply:
No, this is not a known issue in the generator. I'll file an issue to add it back under the correct circumstances.
Tracked by https://github.com/googleapis/gapic-generator-python/issues/565